app/aggregates/deciding/commands/set_funding_decision.rb

```diff
@@ -5,7 +5,28 @@ class SetFundingDecision < Command
 
     def call
       with_decision do |decision|
-        decision.set_funding_decision(user_id:, funding_decision:)
+        decision.set_funding_decision(
+          user_id:, funding_decision:, overall_result:
+        )
+      end
+    end
+
+    private
+
+    # Store the overall results string as it would appear in eForms.
+    def overall_result
+      Maat::OverallResultTranslator.translate(maat_funding_decision)
+    end
+
+    # Since Crime Review only creates Non-means applications, we
+    # can infer that the IoJ has failed when funding_decision is
+    # refused.
+    def maat_funding_decision
+      case funding_decision
+      when /granted/
+        'GRANTED'
+      when /refused/
+        'FAILIOJ'
```

Comment:
Decisions added on Crime Review need a MAAT funding_decision constant in order to be translated into an eForms overall result string.